tests/unit/test__helpers.py

```diff
@@ -466,6 +466,27 @@ def test_w_float_str(self):
 
         self.assertEqual(self._callFUT(value_pb, field_type), expected_value)
 
+    def test_w_float32(self):
```

Comment:
As per FLOAT32 design, the request from client is sent as `number_value` field. What will happen when the input is `float("-inf")` or `float("nan")` and the column in table is of type FLOAT32?

Reply:
We need to verify both insert and read with this data when column type is FLOAT32.
For read we already have an integration test for FLOAT64 `test_execute_sql_w_float_bindings_transfinite `. We can copy it for FLOAT32.

Reply:
behaviour for `float("-inf")` or `float("nan")` is same as FLOAT64, added the skipped test case for manual verification.